src/main/utils.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * PSP utility functions
+ *
+ * A set of utility function used throughout the components used in the main process of the PSP.
+ * Copyright 2016 Steven Githens
+ * Copyright 2016-2017 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+"use strict";
+
+var os       = require("os");
+var fluid    = require("infusion");
+var electron = require("electron");
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.app");
+
+/**
+ * Returns whether the underlying OS is Windows 10 or not.
+ * @return {Boolean} `true` if the underlying OS is Windows 10 or
+ * `false` otherwise.
+ */
+gpii.app.isWin10OS = function () {
+    var osRelease = os.release(),
+        delimiter = osRelease.indexOf("."),
+        majorVersion = osRelease.slice(0, delimiter);
+    return majorVersion === "10";
+};
+
+/**
+* Get the position of `Electron` `BrowserWindows`
```

Comment:
This could do with some further explanation/renaming - it looks like it gets the *desired* window position given its dimensions